nginx.conf

```diff
@@ -0,0 +1,41 @@
+events { }
+http {
+
+  proxy_read_timeout         300s;
+  proxy_send_timeout         300s;
+
+  server {
+    listen 80                default_server;
+    listen [::]:80           default_server;
+    server_name              _;
+    return                  301 https://$host$request_uri;
+  }
+
+  server {
+    listen 443               default_server;
+    server_name              {{HOSTNAME}};
+    ssl                      on;
```

Comment:
Similarly with OCSP stapling and ssl_ciphers. Are defaults finally good enough? It would be nice to write this once and use it forever and get an "A+" or close to it with SSL Labs, or similar.